packages/react-icon-provider/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/react-icon-provider
+
+**React Icon Provider components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```

Comment:
Can you update this text (and the part below) to more accurately reflect the purpose of the package?